### DISCUSSION TITLE
Audio crackling on MacBook Speakers

### DISCUSSION THREAD

**Discussion by smallketchup82** [ppy/osu#27484] [Q&A]
### Type

Other

### Bug description

When using lazer on my 14" Macbook Pro, I notice audio crackling at random moments while the game is open. It only seems to occur when using my MacBook's speakers and doesn't occur when using wired headphones.

The issue seems to only occur when osu! is open. Crackling does not occur when playing media or other files while the game is closed, so I doubt it's an issue with my hardware. Although it also seems to affect my entire system audio but again only when osu! is open. Adjusting system volume while osu! is open results in a crackle (if you have the beep when adjusting audio setting turned on in system settings). Closing osu fixes the issue entirely. I've tried restarting, but the issue persists after restart but only when osu! is open.

Laptop: MacBook Pro 14" Nov 2023 M3 running Sonoma 14.3.1

Screen recordings don't seem to capture it happening. I could try to take a recording with my phone if that might help.

### Version

2024.302.1

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14471475/compressed-logs.zip)

**Comment by frenzibyte** [comment]
Are you running on windowed, borderless, or fullscreen mode?

**Comment by smallketchup82** [comment]
> Are you running on windowed, borderless, or fullscreen mode?

Borderless, which is the default

**Comment by frenzibyte** [comment]
I'm seeing something similar to this. Do you have game mode on? If so, try disabling it and see how it behaves. Also try and see if you get the same issue on windowed mode.

I have converted this thread to discussion until something certain is deduced here.

**Reply by smallketchup82** [reply]
Turning game mode off does seem to fix it. It also seems to get fixed when running with windowed mode (which would also turn off game mode)
